### ISSUE TITLE
Make properties pages more readable

### ISSUE THREAD

**Issue by adamdewey** [msupply-foundation/open-msupply#5158]
## Is your feature request related to a problem? Please describe 👀

The asset and store properties pages could do with a little TLC to make them easier to read. 

Current layout: 

![telegram-cloud-photo-size-1-5136549469339364971-y](https://github.com/user-attachments/assets/03253334-3b62-43e8-80e2-9bdbd7141265)

![image](https://github.com/user-attachments/assets/629dd922-c852-4adb-9acb-9c6a9a8b0b35)

## Describe the solution you'd like 🎁

Ideal layout: 

![telegram-cloud-photo-size-1-5139088185853324788-y](https://github.com/user-attachments/assets/074335b6-babd-4b3d-982e-fc021bf755f8)

### Moneyworks Jobcode 🧰

CCEIDEV

**Comment by mark-prins** [comment]
Yes! would like to have this in place. assigning nice to have, as this is a UI / layout concern

**Comment by ling-msupply** [comment]
Noted

**Comment by rukmal-msupply** [comment]
closing/ not required.
